wordpress-mobile/WordPress-iOS
# Onboarding Improvements: Epilogue screen UI

Part of #17385

This PR includes redesigned UI for Login Epilogue screen.

## Note
- Making the sites tappable / hooking up the create a new site button will be addressed in a subsequent PR
- Fixing the failing UI tests will be addressed in a subsequent PR

## How to test

### Test variation 1
1. Log in with user that has less than 4 sites
2. Confirm UI matches new design

### Test variation 2
1. Log in with user that has more than 4 sites
2. Confirm UI matches new design

### Screenshots
App | Variation 1 | Variation 2
-- | -- | --
WordPress | <img src="https://user-images.githubusercontent.com/6711616/139717698-b8c11993-a434-4f5a-bca1-f011dea51025.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/139717704-79a453a6-0cca-483a-88ff-ded97ffc5c28.png" width=200>
Jetpack | <img src="https://user-images.githubusercontent.com/6711616/139717753-cf4ae707-9209-479a-bf8a-178eec0a8df4.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/139717760-c96bbfe8-fd78-4865-902b-5ded9e3a96f5.png" width=200>



### Regression Notes
1. Potential unintended areas of impact
Jetpack -- The Create A New Site button should be hidden

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested the above steps, see the screenshots

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
